docs/src/Callout.doc.js

```diff
@@ -267,7 +451,53 @@ function Example(props) {
     </Box>
   );
 }
-`}
+        `}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Dismissible"
+      description={`
+        A \`dismissButton\` can be used when a Callout does not indicate a persistent state. This will most commonly be used in info Callouts.
+
+        Do not use dismiss buttons in the following cases:
+
+        - There is a persistent account or page status that the user must address.
+        - The user must access this information again in order to perform a task.
```

Comment:
not strong about it but could we make a little bit more explicit what  `this information` means?